public/app/features/serviceaccounts/state/actions.ts

```diff
@@ -36,18 +37,17 @@ export function loadServiceAccountTokens(saID: number): ThunkResult<void> {
 export function loadServiceAccounts(): ThunkResult<void> {
   return async (dispatch) => {
     try {
-      const response = await getBackendSrv().get(BASE_URL);
+      const response = await getBackendSrv().get(BASE_URL, accessControlQueryParam());
       dispatch(serviceAccountsLoaded(response));
     } catch (error) {
       console.error(error);
     }
   };
 }
 
-export function updateServiceAccount(saID: number): ThunkResult<void> {
+export function updateServiceAccount(serviceAccount: ServiceAccountDTO): ThunkResult<void> {
   return async (dispatch) => {
-    // TODO: implement on backend
-    await getBackendSrv().patch(`${BASE_URL}/${saID}`, {});
+    await getBackendSrv().patch(`/api/org/users/${serviceAccount.id}`, { role: serviceAccount.role });
```

Comment:
:information_source: we should make an update route for service accounts and block off SA editing in users endpoint